assets/i18n/hi.json

```diff
@@ -1,5 +1,42 @@
+	"I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM": "ओपिया डेवलपमेंट टीम, फीडबैक, विचार, सहायता और सुझावों के लिए भी आभारी है, Alex Kauffmann, Allison Barros, Amy Latten, Brett Barros, Crystal Kwok, Daniel Hernandez, Divya Siddarth, Ilwon Yoon, Jennifer Chen, John Cox, John Orr, Katie Berlent, Michael Wawszczak, Mike Gainer, Neil Fraser, Noah Falstein, Nupur Jain, Peter Norvig, Philip Guo, Piotr Mitros, Rachel Chen, Rahim Nathwani, Robyn Choo, Tricia Ngoon, Vikrant Nanda, Vinamrata Singal और  Yarin Feigenbaum की। ",
+	"I18N_ABOUT_PAGE_CREDITS_THANK_TRANSLATEWIKI": "हम भीड़-भाल अनुवादों को प्रदान करने के लिए <a href=\"https://translatewiki.net/wiki/Translating:oppia\">translatewiki.net</a> का शुक्रिया अदा करना चाहेंगे।",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_DONATE_BUTTON": "दान करें",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_GET_INVOLVED_BUTTON" : "शामिल हों",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_HEADING": "ओपिया फाउंडेशन",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_1": "ओप्पीा की वेबसाइट और सोर्स कोड ओपिया फाउंडेशन द्वारा समर्थित हैं, ज्यों की कैलिफोर्निया राज्य में पंजीकृत कर-मुक्त 501 (सी) (3) गैर-लाभकारी संगठन है।",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_2": "यह फाउंडेशन दुनिया भर के योगदानकर्ताओं और दाताओं के उदार सहयोग पर निर्भर करता है जिससे वे किसी भी मज़ेदार और प्रभावी तरीके से कुछ भी जानने के लिए सशक्त बनाने के अपने मिशन की ओर काम कर सकें। ",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_3": "यदि आप इन प्रयासों के समर्थन में सैकड़ों अन्य व्यक्तियों में शामिल होना चाहते हैं, तो ओपिया फाउंडेशन को <a href=\"/donate\">दान करने</a> या <a href=\"/contact\">अन्य तरीकों से शामिल </a>होने के बारे में अधिक जानें।",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4": "फाउंडेशन के निदेशक अमित ड्यूश, जेकब डेविस, और शॉन लिप हैं। फाउंडेशन के <a href=\"https://drive.google.com/folderview?id=0B9KSjiibL_WDfjlpUUpqampZSTBOTmRqM2IyaGN3R3NlVWstZlhKcjlZQ3VMTHJBQWNGTFU&usp=sharing\" class=\"inline-links\">बायलॉज़ और मिनट </a>पढ़ने के लिए उपलब्ध हैं। यदि आप फाउंडेशन से संपर्क करना चाहते हैं, तो कृपया ईमेल करें: <a href=\"mailto:admin@oppia.org\" class=\"inline-links\">admin@oppia.org</a>।",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_4_HEADING": "निदेशक",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5": "ओपिया के अन्वेषणों की सभी सामग्री <a href=\"https://creativecommons.org/licenses/by-sa/4.0/legalcode\" class=\"inline-links\">सीसी-बाय-एसए 4.0 </a>के तहत लाइसेंस की आवश्यकता के छूट (विशेष रूप से, धारा 3 (ए) (1) और 3 (ए) (2)) के अंतर्गत लाइसेंस प्राप्त है। यदि आप इस साइट से सामग्री का पुन: उपयोग करते हैं, तो हम आपको प्रासंगिक अन्वेषण पृष्ठ के लिंक को शामिल करने के लिए प्रोत्साहित करते हैं, लेकिन इसकी आवश्यकता नहीं है।",
+	"I18N_ABOUT_PAGE_FOUNDATION_TAB_PARAGRAPH_5_LICHEADING": "लाइसेंस",
```

Comment:
hmm, this is kind of puzzling. It'll be really good if you could look into it again -- I believe there should be a test that checks that all the keys are in the qqq file. Could you check that it's actually being run, etc?